l10n/en/mozorg/home-m24.ftl

```diff
@@ -0,0 +1,106 @@
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/
+
+# HTML Page title
+m24-home-page-title = Internet for people, not profit — { -brand-name-mozilla }
+
+# HTML page description
+m24-home-page-desc = Did you know? { -brand-name-mozilla } — the maker of { -brand-name-firefox } — fights to keep the internet a global public resource, open and accessible to all.
+
+## Intro
+
+m24-home-welcome-to-mozilla = Welcome to { -brand-name-mozilla }
+m24-home-from-trustworthy-tech = From trustworthy tech to policies that defend your digital rights, we put you first — always.
+m24-home-learn-about-us = Learn about us
+
+## Products
+
+m24-home-love-the-internet = Love the internet again
+m24-home-break-free-from = Break free from big tech — our products put you in control of a safer, more private internet experience.
+m24-home-explore-our-products = Explore our products
+m24-home-firefox = { -brand-name-firefox }
+m24-home-get-the-gold = Get the gold standard for browsing with speed, privacy and control.
+m24-home-thunderbird = { -brand-name-thunderbird }
+m24-home-go-chaosfree-with = Simplify your life with one app for all your emails, calendars and contacts.
+m24-home-fakespot = { -brand-name-fakespot }
+# "fake laughs" is a joke and the ;) is a winking face to hint it's a joke. You can translate this in a away that keeps both the repetition and the joke: fake reviews, fake products, fake ___. Or leave the joke and winking face out
+m24-home-spot-fake-reviews = Spot fake reviews, products and maybe one day, even fake laughs ;)
```

Comment:
**Suggestion: (non-blocking)** this one could be another good contender for en en-US only string, as I'm not entirely sure other languages will get this joke well enough to translate it?

Reply:
✅